src/nbl/video/device_capabilities/device_limits.json

```diff
@@ -910,85 +910,74 @@
-                    "value": false,
-                    "expose": "DISABLE"
+                    "value": false
                 },
                 {
                     "type": "bool",
                     "name": "shaderDenormPreserveFloat32",
-                    "value": true,
-                    "expose": "REQUIRE"
+                    "value": false
                 },
                 {
                     "type": "bool",
                     "name": "shaderDenormPreserveFloat64",
-                    "value": false,
-                    "expose": "DISABLE"
+                    "value": false
                 },
                 {
                     "type": "bool",
                     "name": "shaderDenormFlushToZeroFloat16",
-                    "value": false,
-                    "expose": "DISABLE"
+                    "value": false
                 },
                 {
                     "type": "bool",
                     "name": "shaderDenormFlushToZeroFloat32",
-                    "value": true,
-                    "expose": "REQUIRE"
+                    "value": false
                 },
                 {
                     "type": "bool",
                     "name": "shaderDenormFlushToZeroFloat64",
-                    "value": false,
-                    "expose": "DISABLE"
+                    "value": false
                 },
                 {
                     "type": "bool",
                     "name": "shaderRoundingModeRTEFloat16",
                     "value": false,
-                    "expose": "DISABLE"
+                    "comment": ["ROADMAP2024 but no good support yet"]
                 },
                 {
                     "type": "bool",
                     "name": "shaderRoundingModeRTEFloat32",
-                    "value": true,
-                    "expose": "REQUIRE"
+                    "value": false,
+                    "comment": ["ROADMAP2024 but no good support yet"]
                 },
                 {
                     "type": "bool",
                     "name": "shaderRoundingModeRTEFloat64",
-                    "value": false,
-                    "expose": "DISABLE"
+                    "value": false
                 },
                 {
                     "type": "bool",
                     "name": "shaderRoundingModeRTZFloat16",
-                    "value": false,
-                    "expose": "DISABLE"
+                    "value": false
                 },
                 {
                     "type": "bool",
                     "name": "shaderRoundingModeRTZFloat32",
-                    "value": true,
-                    "expose": "REQUIRE"
+                    "value": false
                 },
                 {
                     "type": "bool",
                     "name": "shaderRoundingModeRTZFloat64",
-                    "value": false,
-                    "expose": "DISABLE"
+                    "value": false
```

Comment:
why mindlessly revert everything and not just stuff that your 3070 doesn't support ?

Also